packages/plugin-hardhat/test/linked-libraries.js

```diff
@@ -105,7 +104,7 @@ async function deployLibrary(libraryName) {
 }
 
 async function getLinkedContractFactory(contractName, libraries) {
-  const cArtifact = await readArtifact(config.paths.artifacts, contractName);
+  const cArtifact = await artifacts.readArtifact(config.paths.artifacts, contractName);
```

Comment:
I don't think the path to artifacts is necessary now. (Based on [Migrating a plugin from Buidler to hardhat](https://hardhat.org/advanced/migrating-buidler-plugin.html).)

```suggestion
  const cArtifact = await artifacts.readArtifact(contractName);
```